### DISCUSSION TITLE
README.md content disappears when deadline added

### DISCUSSION THREAD

**Discussion by howshekilledit** [community/community#72728] [GitHub Education]
### Hi everyone,

So I've had an issue with my last few GitHub Classroom assignments where when the bot makes a commit adding the "Review Deadline" link to README.md, all the instructions in the readme disappear. 

The really strange thing is that this will only happen for ONE STUDENT's repo, and it's a different student every time. 

Any thoughts?

**Comment by bartzalewski** [comment]
Hey @howshekilledit,

Please check if the bot's code correctly appends the "Review Deadline" link rather than overwriting the entire README.md.
